### PR TITLE
collection: Modify for latest yamllint requirements

### PR DESCRIPTION
**Pull request by berndfinger**
ansible-lint 24.7.0 produced:

```
WARNING  Found incompatible custom yamllint configuration (.yamllint.yml), please either remove the file or edit it to comply with:
  - comments.min-spaces-from-content must be 1
  - octal-values.forbid-implicit-octal must be true
  - octal-values.forbid-explicit-octal must be true.
```

This PR ensures that the collection is yamllint clean again.

Relates to issue #810.

### CODE REVIEW

**Review by sean-freeman — APPROVED** [review]
lgtm 👍

File: roles/sap_hana_preconfigure/tasks/sapnote/2684254/configuration.yml [hunk 1]
```diff
@@ -66,15 +66,15 @@
 - name: "Energy Performance Bias (EPB, applies to Intel-based systems only)"
   ansible.builtin.lineinfile:
     path: /etc/init.d/boot.local
-    mode: 0744
+    mode: "0744"
```
Comment (Wabri) [inline roles/sap_hana_preconfigure/tasks/sapnote/2684254/configuration.yml:69]: Why not execution?

**Review by Wabri — COMMENTED** [review]
Just 1 question

File: roles/sap_hana_preconfigure/tasks/sapnote/2684254/configuration.yml (hunk 1 above)
Comment (berndfinger) [reply roles/sap_hana_preconfigure/tasks/sapnote/2684254/configuration.yml:69]: We have mode `0755` for this file defined here:
https://github.com/sap-linuxlab/community.sap_install/blob/c1b479ef710f0f0803a5a081eafd621465c86d4b/roles/sap_hana_preconfigure/tasks/RedHat/generic/disable-ksm.yml#L8-L12

Also from looking at my RHEL test systems, it seems that this is the default for RHEL. So likely it is also the default for SLES.

**Pull request merged**